tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -330,13 +332,16 @@ generate_qemu_options() {
 	# building QEMU.
 	case "$arch" in
 	aarch64)
+		qemu_options+=(functionality:--disable-virtiofsd)
 		;;
 	x86_64)
+		qemu_options+=(functionality:--disable-virtiofsd)
 		;;
 	ppc64le)
 		qemu_options+=(functionality:--enable-virtiofsd)
 		;;
```

Comment:
AFAICT the Rust implemtation also exist for ppc64le. Why do we still need the C version ?

Reply:
AFAIR @Amulyam24 hit some issues with the ppc64le version of virtiofsd, and as I don't have access to different HW platforms, I'm relying on the maintainers to do the switch whenever things are all working for them.

Reply:
Hi @fidencio @gkurz. Yes, that's right. There are some issues with running the rust implementation on ppc64le. The fixes are merged in the official repo. Planning to switch to it soon once the other issues are resolved.